client/js/views/GameView.vue

```diff
@@ -1269,6 +1269,7 @@ export default {
     color: #111111;
     & #history-logs {
       overflow: auto;
+      overflow-wrap: anywhere;
```

Comment:
Resolved issue with overflowing text. 
![image](https://user-images.githubusercontent.com/47369872/206316918-44ffebf3-52be-49e4-b18c-4c3a8b2297d5.png)
![image](https://user-images.githubusercontent.com/47369872/206316937-651111a3-667e-4eac-996c-6e9d8e1bad16.png)
![image](https://user-images.githubusercontent.com/47369872/206316955-d4998808-ca39-4137-8a73-eccecb813a01.png)
![image](https://user-images.githubusercontent.com/47369872/206316980-a0b17f87-fb84-404d-849f-d6a7429fe9dc.png)

Reply:
@peterancourt what do you think?

Reply:
Why not `break-word` instead of `anywhere`? `anywhere` is buck wild and will break a string literally anywhere to make it fit.

Reply:
Break-word isn't working in the flex container. From what I have read this is due to break-word not shrinking the min-content intrinsic size where as anywhere does shrink it. With break word it still overflows.

Reply:
@seriouslysean from what I can gather. break-word will calculate the min content size for width as if no words have been broken. Where Anywhere will calculate the min-content size with the soft breaks included.

From what I have been gathering it looks like on the flex container we could set min-width: 0; overriding the default auto and that should allow us to use break-word and have it work.

Reply:
@seriouslysean if the main issue is long usernames then won't break word not affect them?

Reply:
I didn't consider that as I hadn't tested it locally. I don't think we should be breaking usernames but I guess we kind of have to.

Maybe we eventually move to "Player/Opponent labels and some other way to show which user?

Reply:
@seriouslysean I agree. Necessary evil for the time being and some kind of tokenizing for player/opponent would probably be best in the long run